mmv1/templates/terraform/examples/lustre_instance_basic.tf.tmpl

```diff
@@ -0,0 +1,34 @@
+resource "google_lustre_instance" "{{$.PrimaryResourceId}}" {
+  instance_id = "{{index $.Vars "name"}}"
+  location = "us-central1-a"
+  description = "test instance"
```

Comment:
One of the test configs (here or in the update test) should have optional fields omitted, to verify they are in fact optional.